templates/dataset_description.json

```diff
@@ -1,23 +1,23 @@
 {
```

Comment:
This file does not need to be changed

Reply:
The file had some indentation change as per the python code. I have now uploaded file dataset_description.json as it was before.